lib/widgets/about_us_nossos_valores.dart

```diff
@@ -0,0 +1,91 @@
+import 'package:flutter/material.dart';
+import 'package:zenith_monitor/constants/colors_constants.dart';
+import 'package:zenith_monitor/widgets/standard_app_bar.dart';
+
+class NossosValores extends StatelessWidget {
+  NossosValores({Key? key});
+
+  Widget nossosValoresContainer(String text, String pathToImg, String nomeImg) {
+    return Container(
+      width: 340,
+      height: 94,
+      decoration: BoxDecoration(
+        borderRadius: BorderRadius.circular(15),
+        color: eerieBlack,
+      ),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.start,
+        children: [
+          Expanded(
+            flex: 1,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+              children: [
+                Image.asset(
+                  pathToImg,
+                  color: null,
+                  fit: BoxFit.cover,
+                  width: 37.0,
+                  height: 38.11,
+                  colorBlendMode: BlendMode.dstATop,
+                ),
+                Text(
+                  nomeImg,
+                  style: TextStyle(
+                      color: white, fontSize: 11, fontFamily: 'DMSans'),
+                ),
+              ],
+            ),
+          ),
+          Container(
+            child: VerticalDivider(color: white),
+            height: 49,
+          ),
+          Expanded(
+            flex: 4,
+            child: Text(
+              text,
+              textAlign: TextAlign.center,
+              style: TextStyle(color: white, fontFamily: 'DMSans'),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: StandardAppBar(title: "Nossos Valores"),
+        body: buildMainBody(),
+        backgroundColor: raisingBlack);
+  }
+
+  Center buildMainBody() {
+    return Center(
+      child: GridView.count(
+        crossAxisCount: 1,
+        primary: false,
+        childAspectRatio: 4,
+        padding: EdgeInsets.all(10),
+        crossAxisSpacing: 10,
+        mainAxisSpacing: 10,
+        children: <Widget>[
+          nossosValoresContainer(
+              "Incentivar a produção científica nacional e facilitar o acesso à ciência no ambiente universitário.",
+              "assets/images/figScientist.png",
+              "Ciência"),
+          nossosValoresContainer(
```

Comment:
Nesse trecho o widget nossosValoresContainer foi chamado 3 vezes. Como o texto de cada nossosValoresContainer está apenas nesse arquivo, está ótimo assim. Mas pode ser que futuramente essas informações estivessem em outro arquivo ou banco de dados. Dava pra ter feito outro método que interasse sobre essas informações e retornasse apenas uma lista de nossosValoresContainer prontos. Isso facilitaria para quem fosse adicionar ou alterar esses containers, pois não precisariam alterar o código em si. <br><br>Mas isso é só uma dica gente, está perfeito assim :heart: 
[https://stackoverflow.com/questions/50441168/iterating-through-a-list-to-render-multiple-widgets-in-flutter](url)

Reply:
Segui essa recomendação e criei um loop e alguns arrays pra criar esses containers.

Reply:
Está ótimo Fran. Assim, se a gente decidir separar os textos e o código, pode apenas receber essas listas de strings por parâmetro, além de o código ficar menos repetitivo.